src/Divider/Divider.tsx

```diff
@@ -0,0 +1,37 @@
+import type { ComponentPropsWithoutRef } from 'react';
+import styled, { css } from 'styled-components';
+
+export interface DividerProps extends ComponentPropsWithoutRef<'hr'> {
+  /**
+   * Divider의 종류입니다. 강조의 정도에 차이가 있습니다.
+   */
+  variant?: 'default' | 'strong' | 'disabled';
+  /**
+   * Divider의 길이입니다.
+   */
+  length?: string;
+}
+
+const Divider = ({ variant = 'default', length = '100%', ...props }: DividerProps) => {
+  return <DividerContainer variant={variant} length={length} {...props} />;
```

Comment:
```suggestion
  return <hr color={theme.dividerColors[variant]} width={length} size={size} {...props} />;
```

hr 태그의 기본 속성을 사용해도 좋아보여요
https://developer.mozilla.org/ko/docs/Web/HTML/Element/hr